runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsPlugin.java

```diff
@@ -125,7 +141,15 @@ public void onMenuEntryAdd(MenuEntryAdded menuEntryAdded)
 			int image = -1;
 			Color color = null;
 
-			if (config.drawFriendNames() && player.isFriend())
+			if (config.drawClanCallerNames() && isCaller(player))
```

Comment:
This logic looks duplicated, maybe it could be changed to separate method in PlayerIndicatorsService that both forEachPlayer and onMenuEntryAdd will use